x-pack/plugins/security_solution/common/ecs/memory_protection/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export interface MemoryProtection {
+  cross_session?: boolean;
+  // should this be a enum with 'signature' and '  shellcode_thread' ? are there more options?
```

Comment:
im not sure if this is more a question for the pr or leftover comments, but we'll wanna remove it before merge. Idk much about the context for the field itself

Reply:
leftover comment. Thanks for pointing it.